golang-basic/02-apis.md

```diff
@@ -0,0 +1,15 @@
+### Description
+
+Develop API for news aggregator. This API will collect, process, and
+provide access to news articles from multiple sources, delivering a unified and user-friendly interface for accessing
+diverse news content. (check [01-project-description.md](01-project-description.md) for more details).
+
+As a result of this task, it's expected to have a pull request with a design for an API of your app.
```

Comment:
Should they create a PR after git lecture instead?

Reply:
done